common/src/main/java/tachyon/Constants.java

```diff
@@ -152,8 +152,8 @@
       "tachyon.integration.master.resource.cpu";
   public static final String INTEGRATION_MASTER_RESOURCE_MEM =
       "tachyon.integration.master.resource.mem";
-  public static final String INTEGRATION_YARN_ONE_WORKER_PER_HOST =
-      "tachyon.integration.yarn.one.worker.per.host";
+  public static final String INTEGRATION_YARN_MAX_WORKERS_PER_HOST =
+      "tachyon.integration.yarn.max.workers.per.host";
```

Comment:
`tachyon.max.table.metadata.bytes` and `tachyon.max.columns` are two properties that are perhaps obsolete (so we didn't refactor them in the last release). Other active properties put max/min in the end so if we sort properties alphabetically, they are together :) (not sure how attractive it is, though).